src/components/dev-hub/section-header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { colorMap, fontSize, screenSize } from './theme';
+import { P } from './text';
+
+const SectionHeader = styled('div')`
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+    font-size: ${fontSize.small};
```

Comment:
If we use another `P` I don't think we need this here either but we may need to `styled` that `P` to get the line height we want.